store/mockstore/mocktikv/mvcc_leveldb.go

```diff
@@ -1128,8 +1161,12 @@ func (mvcc *MVCCLevelDB) CheckTxnStatus(primaryKey []byte, lockTS, callerStartTS
 	// written before the primary lock.
 
 	if rollbackIfNotExist {
+		// Write rollback record, but not delete the lock on the primary key. There may exist lock which has
+		// different lock.startTS with input lockTS, for example the primary key could be already
+		// locked by the caller transaction, deleting this key will mistakenly delete the lock on
+		// primary key, see case TestSingleStatementRollbackSecondary in session_test suite for example
```

Comment:
this seems need update comments after rename `TestSingleStatementRollback`, and do we need change this after choose method2 that no longer CheckTxnStatus(rollbackIfNotExist=true)?